styles/globals.css

```diff
@@ -42,4 +42,41 @@ a {
   position: absolute;
   top: -11px;
   z-index: 55;
+}
+
+.slick-prev {
+  left: 0 !important;
+  color: #ccc !important;
+  z-index: 5;
+}
+
+.slick-next {
+  right: 0 !important;
+  color: #CCC !important;
+  z-index: 5;
+}
+
+.slick-dotted.slick-slider {
+  margin-bottom: 10px !important;
+}
+
+.slick-dots li button:before {
+  font-size: 12px !important;
+}
+
+.slick-dots li.slick-active button:before {
+  color: #f7d929 !important;
+}
+
+.slick-dots {
+  bottom: 10px !important;
+}
+
+.css-1kmb5dy {
+  transition: all .5s ease-in-out;
+}
+
+.css-b93e9j:hover .css-1kmb5dy {
+  transition: all .5s ease-in-out;
+  transform: rotateY(180deg);
```

Comment:
this will not work 😢

Reply:
Here I Found a Big Problem In chakra ui They Don't Supply [ transform: rotateY(180deg)] or transform at anyway so that's was my only solution to solve it or using jsx and gave it classname.